src/customJs/payuButtonTool/PayuButtonTool.js

```diff
@@ -0,0 +1,35 @@
+import {intl} from "../localization";
+
+const React = window.unlayer.React;
+import PropTypes from 'prop-types';
+
+const baseUrl = 'https://cdn.fromdoppler.com/unlayer-editor/assets';
+
+export const PayuButtonTool = (props) => {
+  const { size, payu_detail, alignment } = props.values;
+  const buttonImgWidth = size === 'small' ? 116 : size === 'medium' ? 145 : 182;
+
+  return (
+      <div style={{
+        display: 'flex',
+        justifyContent: alignment
+      }}>
+        <a role="link" href={payu_detail ? payu_detail : ''} target="_blank">
+          <img
+              style={{ width: buttonImgWidth }}
+              src={`${baseUrl}/${intl.formatMessage({ id: `payu_${size}_button` })}`}
```

Comment:
Que opinás de descartar `baseUrl` y dejar todo dentro de los archivos de las traducciones?

```suggestion
              src={`${intl.formatMessage({ id: `payu_${size}_button_image_URL` })}`}
```

Reply:
Me parece bien, no tengo una preferencia para eso.